### DISCUSSION TITLE
靠AI分析路口衝突熱點

### DISCUSSION THREAD

**Discussion by henrywen547** [1101-GS4719-C/Python-Community-GS4719-C#566] [General]
雖然網路上是說交通部打臉自己訂的機車待轉的規則
這裡就不討論台灣的道路設計好壞了
或許現在的道路設計存在許多瑕疵，但從重新規範到落實，以及硬體設備的改善完成會需要很長的時間
至少我們可以靠這個技術，了解馬路上存在的一些危險的地方及原因，進而去避免長時間待在危險範圍之中
[交通部透過無人機與AI分析路口衝突熱點，結果機車待轉區被認定「超危險」](https://www.techbang.com/posts/95650-ministry-of-transport-use-of-drones-to-analyze)

**Comment by Sazmafuji** [comment]
期待以後能夠依靠AI管理交通，減少道路阻塞

**Comment by PoooYen** [comment]
期待之後AI可以透過無人機指揮交通，開罰違規車主，還有未來在行車安全的情況下台灣能提高限速。

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望以後能靠AI處理交通問題。

**Comment by HuanYuuuuuu** [comment]
台灣的交通設計的確沒有很完善，但最主要還是用路人要注意看路...

**Comment by Ben6uo** [comment]
可不可以用AI管理一下那些常常違規的三寶

**Comment by ghost** [comment]
我是不覺得有這種東西會減少智障的智障行為啦w

**Comment by yi-1111** [comment]
看來還是自己多多注意為上策

**Comment by Jay-722** [comment]
我想不到有什麼方法可以限制那些無極限的三寶

**Comment by sean1325418** [comment]
以後應該都是用AI科技執法了，希望可以防止一些三寶

**Comment by cordydeng** [comment]
AI已經變成我們生活中的一部分了，如果這個技術能夠更成熟，對於交通安全無疑是一大福音

**Comment by Rosancl** [comment]
推樓主，即便這項技術非常成熟，但重新設計規劃各道路設計，再到實際落實會需要很長的時間，
但起碼民眾也可以藉由這個技術，再更注意這項技術分析出的特別危險的地方，進而減少車禍風險。

**Comment by yucheng1028** [comment]
這似乎能解決不少都市塞車問題

**Comment by Lynn12378** [comment]
我覺得很棒，可以提醒駕駛放放速度注意路況，減少交通事故

**Comment by Jimmy911218** [comment]
如果之後出現ai管理一定能對交通產生不錯的引響

**Comment by Hchejxnc** [comment]
藉由AI的幫助，相信之後規劃道路能夠更加完善
